.github/CODEOWNERS

```diff
@@ -1,4 +1 @@
-* @Zejnilovic
-* @dk1844
-* @benedeki
-* @lsulak
+* @Zejnilovic @dk1844 @benedeki @lsulak
```

Comment:
Why this change? Seems to me one person per line is easier to navigate. 🤔

Reply:
Because it is syntactically wrong, it is a rule per line. Meaning that this is read as 

- Owner of everything is Zejnilovic
- Owner of everything is dk1844
- Owner of everything is benedeki
- Owner of everything is lsulak

And the rules are overwriting each other, with the latest rule having priority. So @lsulak being the only code owner.

This is nice for custom rules but not for `*` rules. 

Example of a smidge more complex `CODEOWNERS`
```
# Everything is owned by all of these guys
* @Zejnilovic @dk1844 @benedeki @lsulak

# Except for this, the test part is owned by miroslavpojer
spark-commons-test* @miroslavpojer
```

Reply:
<img width="345" alt="image" src="https://user-images.githubusercontent.com/17467768/212056011-218487ad-7f84-4f30-8f1a-0cd0fa1aefb6.png">

See the current state of this PR. Only lsulak is owner having the little shield next to his name.

Reply:
Thanks for the explanation 🙇